### PR TITLE
applications: asset_tracker_v2: combine cloud provider section

### PR DESCRIPTION
**Pull request by coderbyheart**
This moves the information about the supported cloud providers
closer together and towards the configuration section.
It adds a paragraph about mandatory configuration options.

Signed-off-by: Markus Tacker <Markus.Tacker@NordicSemi.no>

### CODE REVIEW

**Review by jtguggedal — APPROVED** [review]

**Review by jhn-nordic — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

File: applications/asset_tracker_v2/README.rst [hunk 1]
```diff
@@ -226,17 +210,29 @@ These options can be used to enable and disable modules and modify their behavio
...
+
+`Azure support for Asset Tracker v2`_ is currently under implementation.
+
+Setting up the Asset Tracker Cloud Example for AWS
+--------------------------------------------------
+
+The application is compatible with the `Asset Tracker Cloud Example for AWS`_, which is an open source reference implementation of a serverless backend for an IoT product.
 To set up the application to work with the Asset Tracker Cloud Example for AWS, see the `Getting started guide for Asset Tracker Cloud Example for AWS`_.
 
+Once you have finished the setup, provide its broker hostname to the firmware using the :option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` Kconfig option.
```
Comment (umapraseeda) [inline applications/asset_tracker_v2/README.rst:228]:
```suggestion
Once you have finished the setup, you must provide the broker hostname to the firmware using the :option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` Kconfig option.
```

File: applications/asset_tracker_v2/README.rst [hunk 2]
```diff
@@ -226,17 +210,29 @@ These options can be used to enable and disable modules and modify their behavio
...
+Configure the secTag using the :option:`CONFIG_AWS_IOT_SEC_TAG` Kconfig option, e.g. use the example's default of ``42``.
```
Comment (umapraseeda) [inline applications/asset_tracker_v2/README.rst:229]:
```suggestion
Configure the secTag using the :option:`CONFIG_AWS_IOT_SEC_TAG` Kconfig option. For example, you can use the default value of ``42`` that is used in the Asset Tracker Cloud Example for AWS .
```

**Comment by coderbyheart** [comment]
Accepted and merged your changes into the PR @umapraseeda!

**Comment by coderbyheart** [comment]
I also change the link to the WIP Azure support to be pointing to https://github.com/nrfconnect/sdk-nrf/pull/3950

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**